projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,130 +2,110 @@
-- Jetpack AI: Update copy of proofread/feedback tool. [#37611]
-- Jetpack Menu: Add Scan submenu to Jetpack to show history scans. [#37459]
-- Pass suggestion from AI inline extensions to `onBlockDone`. [#37636]
-- Remove the `jetpack-identity-crisis` dependency. [#36968]
-- Social: Modify initial state. [#37644]
-- Social: Update upgrade nudges to use the new plan. [#37638]
-- Stats: Deprecate old stats experience. [#37614]
-- Subscriptions: Release subscription overlay and Subscribe block navigation placement. [#37594]
-- Subscriptions: Add setting headings to divide toggles into groups. [#37576]
-- Tiled Gallery: Force image aspect ratio for square and circle styles. [#37627]
-
-## 13.5-a.5 - 2024-05-27
-### Enhancements
-- AI Assistant: Move Paragraph extension to production. [#37495]
-- Newsletter: Add "From" name setting. [#37502]
-- Newsletter: Add "Reply to" name setting. [#37362]
+- AI Assistant: Add AI Assistant to Paragraph, Heading, List, and List Item blocks. [#37495] [#37386] [#37613]
+- Newsletter: Add "From" and "Reply to" name settings. [#37502] [#37362]
 - Social: Add connection management to editor. [#37405]
-- Social: Add width to connection management container. [#37490]
-
-### Improved compatibility
-- Change how Jetpack social connections are stored on the local site. [#37500]
-
-### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
-- AI Assistant: Add auto-scroll to inline extension. [#37481]
-- AI Assistant: Add paragraph inline extension. [#37461]
-- AI Assistant: Fix selected block while using the AI Assistant extension on a nested block. [#37519]
-- AI Assistant: Toggle inline AI input on Ask AI Assistant button. [#37480]
-- AI Featured Image: Add site type and image model to events being tracked on the tool. [#37474]
-- Fix settings endpoint when subscription_options is not modified. [#37190]
-- Deprecated Errors class. [#37451]
-- Jetpack AI: Open upgrade paths in new tab on every upgrade entry point. [#37457]
-- Jetpack_Currencies: Fix PHP8 fatal error in format_price when passing a string in number_format [#37530]
-- Pay with PayPal: Ensure prices are correctly formatted. [#37534]
-- Sharing button: Increase performance on p2020 theme. [#37497]
-- Social: Update the initial state fields for WP.com. [#37532]
-- Subscriptions: Add name to the overlay group block. [#37574]
-- Subscriptions: Do not clear the subscriber email when current user is empty. [#37485]
-- Subscriptions: Subscription overlay fixes. [#37503]
-- Updated package dependencies. [#36964]
-
-## 13.5-a.3 - 2024-05-20
-### Enhancements
-- AI Assistant: Enable inline Heading extension. [#37386]
+- Social: Add the new Jetpack Social plan. [#37619]
 - Subscribe block: Add button-only style. [#37341]
-- Subscribe block: Allow in the Navigation block. [#37439]
+- Subscribe block: Allow to be placed inside the Navigation block. [#37439]
```

Comment:
Hmm, just looking at previous releases, let's make this `Subscribe Block:` and the one above too.